### ISSUE TITLE
Disabling Radio Buttons

### ISSUE THREAD

**Issue by omgUFOs** [cheat-engine/cheat-engine#1279]
I'm having issues with Cheat Engine disabling my active cheats and enabling other cheats I didn't activate. This is happening more and more frequently...

**Comment by cheat-engine** [comment]
are hotkeys assigned to the affected cheats?

**Comment by omgUFOs** [comment]
Wow that was fast! Yes, numbers are assigned to them.

**Comment by cheat-engine** [comment]
any chance you're hitting the hotkeys by accident?

**Comment by omgUFOs** [comment]
No, I don't need the numbers for what I'm doing. I use a macro recorder and that only uses letters and space. No numbers.
